src/display.jl

```diff
@@ -205,13 +206,32 @@ Allowable formats depend on the backend;
 Resolution can be specified, via `save("path", scene, resolution = (1000, 1000))`!
 """
 function FileIO.save(
-        f::FileIO.File{F}, scene::Scene;
-        resolution = size(scene)
-    ) where F
+        f::FileIO.File, scene::Scene;
+        resolution = size(scene), kwargs...
```

Comment:
Can we write out the expected kwargs and document them?

Reply:
they differ for each backend, though, so they would end up in those backends? for png in Cairo we need `px_per_unit` and for svg/pdf `pt_per_unit`

Reply:
`unit` then refers to the scene "pixels" we currently have

Reply:
Well, even more important to document them here!
This is still the function people call and will call help/autocomplete on, if they dont understand how to use it, no?
Also, cant we just use pt?
`In typography, the point is the smallest unit of measure` so for a pixel based backend, this means pt==pixel, no?

Reply:
Btw, I have a pretty big pet peeve with `kw...` if its not 100% used for a function that just passes them through and has no way to actually write them out. 
Its so hard find out with multiple dispatch, where those keyword arguments actually land and therefore really hard to use them and figure out what is allowed and what not...
You will need a debugger to actually find where those keyword arguments are ending up - which gets even harder, when they're transferred to the IO object :D

Reply:
I agree, but do you have a better alternative? We need arbitrary keywords being able to end up at our backend functions, and different keywords are needed for different backends.

Reply:
And no, we can't call the scene units pt unless we completely lock that conversion in. But in GLMakie one unit is obviously a pixel right now, not a pt. And in pdf / svg pt has a quite specific meaning.

Reply:
>But in GLMakie one unit is obviously a pixel right now, not a pt

I interpreted that pt is the smallest unit of the drawing "backend" which is pixels for GLMakie - or did I misinterpret the meaning of pt from wikipedia? :D 

>We need arbitrary keywords being able to end up at our backend functions

No, we need to have a well defined set of keyword arguments that end up in different backends... This is exactly the point where we should document those differences for different backends :)

Reply:
If you think the pt point of mine is just too confusing because of pre-existing concepts, or because i misunderstood, I won't insist on this :D 
I was just looking at the definition and thought, that's exactly what we would want.

Reply:
A point is a physical measure, it's generally 1/72th of an inch. I wouldn't tie myself to that but instead use an arbitrary "base unit" that is then converted to a meaningful base unit for the backend. Even if that conversion just ends up as 1:1. In CairoMakie it's important to differentiate what it means to have a linewidth of 1, in pdf it's 1pt without any intervention and in png it's 1px.

Reply:
But it's easy to imagine different keywords to configure a Cairo png backend vs a Cairo svg backend, vs a GR backend etc no?

I have an idea, maybe each backend offers objects for this called PngConfig, SvgConfig, etc and the user passes one of those to the save function. Then those configs could have a limited set of allowed keywords

Reply:
ok fair enough....